.github/workflows/checks.yaml

```diff
@@ -366,6 +369,17 @@ jobs:
         run: |
           curl -L "$LINK/cargo-c-x86_64-unknown-linux-musl.tar.gz" |
           tar xz -C $HOME/.cargo/bin
+      - name: Install bzip2
+        uses: awalsh128/cache-apt-pkgs-action@latest
```

Comment:
libbz2-dev + abigailt-tools + all deps is only 2MB compressed. Using `apt install` without cache likely isn't any slower, but does somewhat reduce the risk of a supply chain attack.